blocks_vertical/procedures.js

```diff
@@ -317,3 +317,16 @@ Blockly.Blocks['argument_reporter_string_number'] = {
     });
   }
 };
+
+Blockly.Blocks['procedures_mutator_root'] = {
+  /**
+   * Block for calling a procedure with no return value.
```

Comment:
This comment doesn't seem right?

Reply:
Yeah.  In my next PR I'll update it to "The root block in the procedure editing workspace."

The extensions are also wrong--right now this is just a minimal placeholder block.